cyclops-ui/src/components/k8s-resources/DaemonSet.tsx

```diff
@@ -27,8 +27,8 @@ const DaemonSet = ({ name, namespace }: Props) => {
             group: `apps`,
             version: `v1`,
             kind: `DaemonSet`,
-            name: name,
-            namespace: namespace,
+            name,
+            namespace,
```

Comment:
Can you revert this? At least for me, its easier to see what is passed since other fields are still `group: apps`

Reply:
done